apps/mochi-web/components/profile-dropdrown.tsx

```diff
@@ -27,13 +27,15 @@ export default function ProfileDropdown() {
   return (
     <DropdownMenu>
       <DropdownMenuTrigger asChild>
-        {isLoggedIn && me && (
-          <ProfileBadge
-            avatar={me?.avatar || '/logo.png'}
-            name={truncateWallet(me.profile_name) || 'unknown'}
-            platform={me.platformIcon || '/logo.png'}
-          />
-        )}
+        <div>
```

Comment:
Why do we need a div to avoid the break? IMO, This proves the instability of the component.